_summaries/beltagy2020longformer_2.md

```diff
@@ -0,0 +1,146 @@
+        * Results: State-of-the-art. Small model results on the left and large model results on right.
+
+    <p align="center">
+        <img src="https://d3i71xaburhd42.cloudfront.net/925ad2897d1b5decbea320d07e99afa9110e09b2/5-Table2-1.png" width="30%" height="30%" style="padding-right: 2%;">
+        <img src="https://d3i71xaburhd42.cloudfront.net/925ad2897d1b5decbea320d07e99afa9110e09b2/5-Table3-1.png" width="30%" height="30%">
+    </p>
+
+    * **Masked LM for Pre-training**
+        * Dataset: Corpus of long documents from sources such as Books corpus ([Zhu et al., 2015](https://arxiv.org/abs/1506.06724)) and English Wikipedia. Refer to Appendix C in the paper.
+        * Evaluation: BPC.
+        * Results: Better than RoBERTa for both small and large models, which possibly benefitted from the longer context.
+
+    <p align="center">
+        <img src="beltagy2020longformer/t5.png" width="30%" height="30%">
+    </p>
+
+    * **Fine-tuned Tasks**
+        * Datasets:
+            * Question Answering: WikiHop ([Welbl et al., 2018](https://arxiv.org/abs/1710.06481)), Wikipedia setting in TriviaQA ([Joshi et al., 2017](https://arxiv.org/abs/1705.03551)), distractor setting in HotpotQA ([Yang et al., 2018](https://arxiv.org/abs/1809.09600)).
+            * Coreference Resolution: OntoNotes ([Pradhan et al., 2012](https://aclanthology.org/W12-4501/)).
+            * Document Classification: IMDB sentiment classification ([Maas et al., 2011](https://aclanthology.org/P11-1015/)), Hyperpartisan news detection ([Kiesel et al., 2019](https://aclanthology.org/S19-2145/)).
+        * Evaluation: TriviaQA, Hyperpartisan metrics are F1. WikiHop and IMDB use accuracy. HotpotQA is join F1. OntoNotes is average F1.
+        * Results: The base models did better than RoBERTa's base model. Large models outperforms SoTA for WikiHop and TriviaQA at time of submission, and is off by 1 point from SoTA in HotpotQA at time of submission. Small model results in the top table and large model results in the bottom table.
+
+    <p align="center">
+        <img src="https://d3i71xaburhd42.cloudfront.net/925ad2897d1b5decbea320d07e99afa9110e09b2/8-Table8-1.png" width="50%" height="50%">
+        <br>
+        <img src="beltagy2020longformer/t8.png" width="30%" height="30%">
+    </p>
+
+    * **LED**
+        * Dataset: arXiv summarization dataset ([Cohan et al., 2018](https://aclanthology.org/N18-2097/)), which has 14.5K tokens at the 90th percentile of document lengths.
+        * Evaluation: ROUGE-1 (R-1), ROUGE-2 (R-2), and ROUGE-L (R-L) ([Lin, 2004](https://aclanthology.org/W04-1013/)).
+        * Results: State-of-the-art. Outperforms BigBird ([Zaheer et al., 2020](https://papers.nips.cc/paper/2020/hash/c8512d142a2d849725f31a9a7a361ab9-Abstract.html)).
+
+    <p align="center">
+        <img src="beltagy2020longformer/t11.png" width="30%" height="30%">
+    </p>
+
+## TL;DR
```

Comment:
Great summary. You provide an in-depth look at the technical details of Longformer as well as present the results in an easily digestible manner.